src/Uno.UI/UI/Xaml/Documents/LanguageTagCache.skia.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using Windows.Globalization;
+
+#nullable enable
+
+namespace Windows.UI.Xaml.Documents;
+
+internal class LanguageTagCache
+{
+	[ThreadStatic] private static readonly Dictionary<string, string[]> _languageTagsCache = new();
```

Comment:
Why does this have to be ThreadStatic?

Reply:
Because, as far as I know, there can be multiple UI Threads and each one could have a different default culture.

Reply:
If we consider multi-threading, shouldn't this be a concurrent dictionary?

Reply:
Makes sense, though I think it's very weird we need the language to render `Run`s. I think culture shouldn't affect how `Run`s are rendered.